### PR TITLE
lcab 1.0b12 (new formula)

### PR DESCRIPTION
**Pull request by bfontaine**
[Lcab](http://ohnopub.net/~ohnobinki/lcab/) is a tool to create Cabinet files (`*.cab`). I used it to write a test for Cabextract (#36006).

### CODE REVIEW

File: Library/Formula/lcab.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Lcab < Formula
+  homepage "http://ohnopub.net/~ohnobinki/lcab/"
+  url "ftp://mirror.ohnopub.net/mirror/lcab-1.0b12.tar.gz"
```
Comment (mikemcquaid) [inline Library/Formula/lcab.rb]: Is that `beta 12` is there a stable version?

File: Library/Formula/lcab.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test").write "a test"
+
+    system "#{bin}/lcab", "test", "test.cab"
+    assert File.exist? "test.cab"
```
Comment (mikemcquaid) [inline Library/Formula/lcab.rb]: :+1:

File: Library/Formula/lcab.rb (hunk 1 above)
Comment (bfontaine) [inline Library/Formula/lcab.rb]: Ah yes, this `b` looks like it’s a beta version but I can’t find any other download.

Comment (mikemcquaid) [inline Library/Formula/lcab.rb]: No worries then.

**Pull request closed without merging**